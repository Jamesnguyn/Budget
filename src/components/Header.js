import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Portfolio</h1>
    <NavLink to="/" activeClassName="is-active" exact={true}>HOME </NavLink>
    <NavLink to="/portfolio" activeClassName="is-active">PORTFOLIO </NavLink>
    <NavLink to="/contact" activeClassName="is-active">CONTACT </NavLink>
  </header>
);

export default Header;