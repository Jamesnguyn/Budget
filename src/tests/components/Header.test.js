import React from 'react';
import Header from '../../components/Header';
import ReactShallowRenderer from 'react-test-renderer/shallow';

test('render header', () => {
  const renderer = new ReactShallowRenderer;
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
