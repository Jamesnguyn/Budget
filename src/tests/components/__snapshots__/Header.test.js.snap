// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/dashboard"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
    add expense 
  </NavLink>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
